docs/basics/janusgraph-cfg.md

```diff
@@ -195,7 +195,7 @@ Solr index configuration
 | Name | Description | Datatype | Default Value | Mutability |
 | ---- | ---- | ---- | ---- | ---- |
 | index.[X].solr.configset | If specified, the same solr configSet can be reused for each new Collection that is created in SolrCloud. | String | (no default value) | MASKABLE |
-| index.[X].solr.dyn-fields | Whether to use dynamic fields (which appends the data type to the field name). If dynamic fields is disabledthe user must map field names and define them explicitly in the schema. | Boolean | true | GLOBAL_OFFLINE |
+| index.[X].solr.dyn-fields | Whether to use dynamic fields (which appends the data type to the field name). If dynamic fields is disabled, the user must map field names and define them explicitly in the schema. | Boolean | true | GLOBAL_OFFLINE |
```

Comment:
This is both a spelling and grammar change. I usually try not to mix the two, but it didn't seem particularly worthwhile to isolate the pair.

Note: the word `is` before this should be `are`, but that's a grammar change and should be done as part of a grammar cleanup of the document as opposed to a spelling pass on the repository.

Reply:
See https://github.com/JanusGraph/janusgraph/pull/1883#discussion_r343381325